src/sass/components/ATATPhoneInput.scss

```diff
@@ -5,7 +5,7 @@
     .v-input__control > {
```

Comment:
please add this to prevent double-border on right of country dropdown
![image](https://user-images.githubusercontent.com/90333349/163856246-e74ea6b1-471e-4c20-8b4b-a38a157709e7.png)

<img width="241" alt="image" src="https://user-images.githubusercontent.com/90333349/163856089-66a8b696-360a-460b-ad2b-5e95b9ba7c9e.png">